config/samples/dataplane_v1beta1_openstackdataplanenodeset_baremetal_with_ipam.yaml

```diff
@@ -0,0 +1,154 @@
+apiVersion: dataplane.openstack.org/v1beta1
+kind: OpenStackDataPlaneNodeSet
+metadata:
+  name: openstack-edpm-ipam
+spec:
+  env:
+    - name: ANSIBLE_FORCE_COLOR
+      value: "True"
+    - name: ANSIBLE_ENABLE_TASK_DEBUGGER
+      value: "True"
+  services:
+    - configure-network
+    - validate-network
+    - install-os
+    - configure-os
+    - run-os
+    - ovn
+    - libvirt
+    - nova
+  deployStrategy:
+      deploy: true
+  baremetalSetTemplate:
+    deploymentSSHSecret: dataplane-ansible-ssh-private-key-secret
+    bmhLabelSelector:
```

Comment:
The current edpm baremetal job failure[1] is because it's missing bmhNameSpace(openstack)[2] and looking for the baremetal nodes in the default `openshift-machine-api` namespace.
[1] 
2023-08-23T21:32:39.261484439+00:00 stderr F 2023-08-23T21:32:39Z	DEBUG	controller-runtime.webhook.webhooks	wrote response	{"webhook": "/validate-baremetal-openstack-org-v1beta1-openstackbaremetalset", "code": 403, "reason": "Unable to find 2 requested BaremetalHosts for scale-up (0 in use, 0 available)", "UID": "abdde861-3961-4eaa-9331-d48c32496d88", "allowed": false}
[1] https://github.com/openstack-k8s-operators/install_yamls/blob/main/devsetup/scripts/gen-edpm-baremetal-kustomize.sh#L61-L63